client/js/lounge.js

```diff
@@ -1331,7 +1331,7 @@ $(function() {
 					} else {
 						var current = $(this);
 
-						if (current.attr("class") === "date-marker") {
+						if (current.hasClass("date-marker") && prev.hasClass("date-marker")) {
```

Comment:
If we have the unread marker at the top, and we have 2 date markers straight after, this will not delete 1 of the date markers, will it? Because it'll drop into the else and stop there?